sstables/mx/writer.cc

```diff
@@ -1006,6 +1040,48 @@ void writer::consume(tombstone t) {
     }
 }
 
+void writer::maybe_record_large_partitions(const sstables::sstable& sst, const sstables::key& partition_key, uint64_t partition_size) {
+    auto& entry = _large_data_stats.map.at(large_data_type::partition_size);
+    if (entry.max_value < partition_size) {
```

Comment:
it's more intuitive to me if it were written if (size > max) max = size, but I am also fine with this :-)